### ISSUE TITLE
Table 2 reproductibility

### ISSUE THREAD

**Issue by davideboscaini** [giuliomattolin/ConfMix#4]
Hi! I'm trying to reproduce the results of Table 2 and I have some questions.
1. You are evaluating the detection quality in terms of mAP, but which IoU threshold are you using? Is it correct to think to your metric as the mAP 0.50 used by pycocotools? Or as mAP are you referring to what pycocotools calls mAP 0.5:0.05:0.9?
2. The data you shared contains gt bbox annotations only for the training set of Cityscapes and for the training and validation set of FoggyCityscapes. Where are you evaluating the results you are showing? On FoggyCityscapes validation set?
3. The data you shared contains annotations only for 2965 out of 2975 training images, i.e. annotations of 10 images are missing. Is it fine?
Thanks.
D

**Comment by giuliomattolin** [comment]
Hi @davideboscaini, 
 
1. In line with the literature, in all our experiments we used as evaluation metric the mAP 0.5.
 
2. Yes, as you correctly pointed out, following the literature we evaluated the results on the target validation set.
 
3. Some images may not have an associated annotation file since they do not contain any object of interest. They will be considered as having an empty annotation file and still be used during training.
 
I hope this will help you, let me know if you have any other questions!
